linera-service/src/cli_wrappers/wallet.rs

```diff
@@ -384,6 +384,7 @@ impl ClientWrapper {
         &self,
         from: ChainId,
         to_public_key: Option<PublicKey>,
+        initial_balance: Option<u128>,
```

Comment:
Why an option? Isn't `None` the same as `0`?

And it should probably be `Amount`, not `u128`.

Reply:
Ah true, the default is `0`, good point
On the `Amount`, what I thought was weird is that `initial_funding` for example is also an `Amount`, but everywhere that `--initial-funding` is used, actual integers are being used. It seems like it gets automagically converted to `Amount` when passed over. And the advantage here IMHO is that when the commands get printed on logs, they'll be easier/more intuitive to read than an `Amount` with the 18 zeros.
What do you think?

Reply:
Chatted offline, seems like my concerns with `Amount` won't actually be a problem, so will be changing these to `Amount`